### ISSUE TITLE
Add option to show status in font without emojis

### ISSUE THREAD

**Issue by b-chu** [ejoffe/spr#357]
When ssh into a remote instance, system fonts don't properly render the bits for status, update, or merge. It would be great to add a config option to display bits that are supported for all fonts.

Sample output below:
```
> git spr status
> git rev-parse --show-toplevel
> github fetch pull requests
> git log --format=medium --no-color upstream/dev..HEAD
> git branch --no-color
[____] https://github.com/mosaicml/composer/pull/2593 : Add Rank Planners
[____] https://github.com/mosaicml/composer/pull/2584 : Update types to follow PEP 585
[____] https://github.com/mosaicml/composer/pull/2582 : Add SavePlanner and LoadPlanner support in Trainer
```

**Comment by ejoffe** [comment]
Use `statusBitsEmojis` in the User config.

**Comment by b-chu** [comment]
I set it as false and it doesn't render properly

**Comment by ejoffe** [comment]
Gotcha, I'll add a basic status bits, these are the current set:
```	asciiCheckmark = "✔"
	asciiCrossmark = "✗"
	asciiPending   = "·"
	asciiQuerymark = "?"
	asciiEmpty     = "-"
	asciiWarning   = "!"
```

**Comment by b-chu** [comment]
I think the current set is extended ASCII? I personally prefer lowercase x v, but both are fine.
https://theasciicode.com.ar/

**Comment by ejoffe** [comment]
Your change released in v0.14.4. Thank you for the contribution!
